code/game/objects/structures/morgue.dm

```diff
@@ -399,7 +399,7 @@ GLOBAL_LIST_EMPTY(crematoriums)
 	name = "morgue tray"
 	desc = "Apply corpse before closing."
 	icon_state = "morguet"
-	pass_flags_self = PASSTABLE
+	pass_flags_self = LETPASSTHROW
```

Comment:
why remove passtable? doesn't this mean things can't crawl under it (like aliens) now?

Reply:
Added passtable back, it was removed by mistake.